lib/ansiblelint/__init__.py

```diff
@@ -101,6 +101,14 @@ def matchtasks(self, file, text):
                            file['path'], self, message))
         return matches
 
+    def _unpack_result(self, play, result):
```

Comment:
Since this method doesn't use `self`, it should be a plain function instead.

Reply:
can do